File: data/src/main/java/home/oleg/placesnearme/data/database/DaoProvider.java
package home.oleg.placesnearme.data.database;

import home.oleg.placesnearme.data.dao.DetailedVenueDao;
import home.oleg.placesnearme.data.dao.DetailedVenueHistoryDao;

public interface DaoProvider {
    DetailedVenueDao getDetailedVenueWithPhotosDao();
    DetailedVenueHistoryDao getVenueHistoryDao();
}

File: core-data/src/main/java/home/oleg/placesnearme/data/database/DaoProvider.kt
package home.oleg.placesnearme.data.database

import home.oleg.placesnearme.data.dao.DetailedVenueDao
import home.oleg.placesnearme.data.dao.DetailedVenueHistoryDao

interface DaoProvider {
    val detailedVenueWithPhotosDao: DetailedVenueDao
    val venueHistoryDao: DetailedVenueHistoryDao
}
